trashtalk/views/users.py

```diff
@@ -76,21 +79,21 @@ def post(user_id):
         db_session.add(user)
         db_session.commit()
         return redirect(url_for("users.get", user_id=user_id),
-                        code=status.HTTP_403_FORBIDDEN)
+                        code=status.HTTP_200_OK)
```

Comment:
Think we had some renegade copy/pasta that ended up with `403_FORBIDDEN` in a few places! Simply changed these responses to send back correct status.